lib/jsdom/living/nodes/HTMLLabelElement-impl.js

```diff
@@ -60,9 +61,28 @@ class HTMLLabelElementImpl extends HTMLElementImpl {
     return null;
   }
 
-  _activationBehavior() {
+  _activationBehavior(event) {
+    // Check if the event's target is an inclusive descendant of any interactive content descendant of this <label>.
+    // If so, do nothing.
+    if (event.target && event.target !== this && isInclusiveAncestor(this, event.target)) {
+      for (const ancestor of domSymbolTree.ancestorsIterator(event.target)) {
+        if (ancestor === this) {
+          break;
+        }
+        if (isInteractiveContent(ancestor)) {
+          return;
+        }
+      }
+    }
+
     const node = this.control;
     if (node && !isDisabled(node)) {
+      // Check if the control is an inclusive ancestor of the event's target (and has already received this event).
+      // If so, do nothing.
```

Comment:
Can you file a spec bug (or PR) for this case? I think the spec should include it.

A bug might be nice because we could discuss different possibilities. E.g. adding another "must do nothing" case, vs. adding this algorithm to the spec (perhaps all browsers follow it?), vs. adding this algorithm to the spec but with the "sendClickToAssociatedNode" step as optional...